lib/containers/basetest.pm

```diff
@@ -28,4 +29,8 @@ sub containers_factory {
     return $engine;
 }
 
+sub test_flags {
+    return get_var('PUBLIC_CLOUD') ? {no_rollback => 1} : {};
+}
+
```

Comment:
i wonder if this makes any difference. the `Base` defines the same `test_flags`. isnt this taken by parent class?

Reply:
You're right. We're missing the `QEMU_DISABLE_SNAPSHOTS` variable.